src/Conf.ml

```diff
@@ -1009,8 +1020,11 @@ let parse_line config ~verbose ~from s =
   | _ -> Error (`Malformed s)
 
 let is_project_root dir =
-  List.exists project_root_witness ~f:(fun name ->
-      Caml.Sys.file_exists (Filename.concat dir name) )
+  match !root with
+  | Some root -> String.equal dir root
```

Comment:
This won't work with
- relative path
- root with a trailing separator
- path containing `./` and `../`

Note that the logic around `.ocamlformat` files lookups does not work in presence of `../` already